app/Models/Language/Language.php

```diff
@@ -357,9 +356,8 @@ public function scopeIncludeCurrentTranslation($query)
         $query->leftJoin('language_translations as current_translation', function ($join) {
             $priority_q = \DB::raw('(select max(`priority`) from language_translations WHERE language_source_id = languages.id LIMIT 1)');
             $join->on('current_translation.language_source_id', 'languages.id')
-            ->where('current_translation.language_translation_id', '=', $GLOBALS['i18n_id'])
-            ->where('current_translation.priority', '=', $priority_q)
-            ->orderBy('current_translation.priority', 'desc')->limit(1);
```

Comment:
It was removed because the `orderBy` and `limit` methods do not take effect inside the join method.